event-app-api/server.js

```diff
@@ -94,14 +136,61 @@ app.get("/users", async (req, res) => {
   }
 });
 
-sequelize
-  .sync({ alter: true })
-  .then(() => {
-    const port = 3000;
-    app.listen(port, () => {
-      console.log(`App is listening on port ${port}`);
+//sending a get request to the events endpoint
+app.get("/events", async (req, res) => {
+  try {
+    const events = await Event.findAll({
```

Comment:
You're querying all events here, not just the one related to the user who's logged in

Reply:
You can put this in your backlog and leave it off this PR as it is very long already